comp/core/agenttelemetry/agenttelemetryimpl/config.go

```diff
@@ -0,0 +1,418 @@
+//                 "Count": 2,
+//                 "Instances": {
+//                   "worker_1": {
+//                     "Utilization": 0.05
+//                   },
+//                   "worker_2": {
+//                     "Utilization": 0.17
+//                   }
+//                 }
+//               }
+//             }
+//           }
+//
+//       ... with "extra" like this ...
+//           ...
+//           extra:
+//             workers.count: '.runnerStats.Workers.Count'
+//             workers.utilization: '[.runnerStats.Workers | .. | objects | .Utilization] | add'
+//
+//       ... the following JSON object will be added to the agent telemetry statys payload ...
+//           "workers": {
+//             "count": 2,
+//             "utilization": 0.22
+//           }
+//
+// profiles[].schedule (optional)
+// --------------------------------
+// Specified when agent telemetry payloads to be generated and emitted. If not specified,
+// configured payloads willbe generated and emitted on the following schedule (the details
+// are described in the comments below.
+//
+//    (legend - 300s=5m, 900s=15m, 1800s=30m, 3600s=1h, 86400s=1d)
+//
+//        schedule:
+//          start_after: 30
+//          iterations: 0
+//          period: 900
+//
+// profiles[].schedule.start_after (optional)
+// ------------------------------------------
+// Number of seconds to wait after agent start before starting telemetry collection for the
+// profile. If not specified, default values are specified above.
+//
+// profiles[].schedule.iterations (optional)
+// -----------------------------------------
+// Number of telemetry collection iterations to perform for the profile. To indicate infinite
+// number of iterations, use 0. If not specified, default value of 0 will be used.
+//
+// profiles[].schedule.period (optional)
+// -------------------------------------
+// Number of seconds to wait between telemetry collection iteration for the profile. If not
+// specified, default values are specified above.
+
+// ----------------------------------------------------------------------------------
+//
+// Default agent telemetry profiles config if not specified in the agent config file.
+var defaultProfiles = `
+  profiles:
+  - name: core-metrics
+    metric:
+      exclude:
+        zero_metric: true
+        tags:
+          - "check_name:cpu"
+          - "check_name:memory"
+          - "check_name:uptime"
+          - "check_name:network"
+          - "check_name:io"
+          - "check_name:file_handle"
+      metrics:  
+        - name: checks.runs
+        - name: checks.execution_time
+        - name: checks.warnings
+        - name: checks.metrics_samples
+        - name: checks.events
+        - name: logs.decoded
+        - name: logs.processed
+        - name: logs.sent
+        - name: logs.network_errors
+        - name: logs.dropped
+        - name: logs.sender_latency
+        - name: logs.destination_http_resp
```

Comment:
This metric has `url` tag that may leak sensitive info. Is that expected?

Since all these metrics are defined elsewhere, it would be easy to add tags with sensitive information by mistake. Would it make sense to allow-list all tags?

Reply:
>  This metric has url tag that may leak sensitive info. Is that expected?

Not sure about danger of url in terms of privacy but honestly I do not know what urls can be appeared there. I presumened it was Datadog domain url. 

> Since all these metrics are defined elsewhere, it would be easy to add tags with sensitive information by mistake. Would it make sense to allow-list all tags?
True, it did concerned me. It certainly has big hole when privacy may be inadvertently violated  by any developer without even realizing it. I will add facility to specify explicit tag management. I did not have time to do it before.

Reply:
I believe that I have addressed concerns in sub-sequent commit by adding to configuration needs to explicitly specify associated tags. If not tags are specified, all of them will be dropped (but timeseries aggregated accordingly)